package.json

```diff
@@ -63,6 +63,7 @@
     "eslint-plugin-import": "^2.13.0",
     "husky": "^0.14.3",
     "jest": "^23.4.1",
+    "jest-cli": "^23.6.0",
```

Comment:
I added this because running `npm test` was complaining that `jest` can not be found (because you need the jest-cli now).